src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -295,4 +295,12 @@ In order to debug this project, add an executable project to this solution which
     <value>The .xaml file was in an unexpected format, near line {0} column {1}.</value>
     <comment>{0} is the line number of the problematic XAML, and {1} is the column.</comment>
   </data>
+  <data name="InfoBarMessageNuGetCycleDetected" xml:space="preserve">
+    <value>A circular dependency A -&gt; B -&gt; A was detected during NuGet Restore which caused NuGet to stop restoring.
+Because the NuGet restore didn't finish correctly, the project might be in a bad state.
+
+This might be caused because one or more projects have a package that sets AssetTargetFallback, not supported for this scenario, which caused certain PackageReference to be included.
+
+NuGet Error NU1108</value>
```

Comment:
I'd recommend removing this. 

NU1108 talks about a restore cycle within a restore graph. This merely exposes a limitation in what VS can do wrt to DT builds. 

The NU1108 text doesn't really talk about any of these issues anyways.

Reply:
@ocallesp were you able to apply @nkolev92's feedback here?

Reply:
This was addressed in ff84ebd58605f6878decb2619c841f4a73e96e7c.